oap-server/server-starter/src/main/resources/application.yml

```diff
@@ -416,6 +416,8 @@ query:
     # Maximum complexity allowed for the GraphQL query that can be used to
     # abort a query if the total number of data fields queried exceeds the defined threshold.
     maxQueryComplexity: ${SW_QUERY_MAX_QUERY_COMPLEXITY:100}
+    # Allow user add, disable and update UI template
+    enableUpdateUITemplate: ${SW_ENABLE_UPDATE_UI_TEMPLATE:true}
```

Comment:
Should we disable this on default? We don't have authentication after all.

Reply:
make sense